src/styles/molecules/_molecules.datepicker.scss

```diff
@@ -83,69 +130,160 @@ $datepicker-unavailable-bg:          $datepicker-bg !default;
   button {
     background-color: transparent;
     border: none;
+    color: var(--datepicker-text-color);
     cursor: pointer;
+    font-size: 1rem;
     height: 100%;
     padding: 0;
     position: absolute;
-    transition: background var(--animation-normal);
+    transition: color var(--animation-normal);
     width: 100%;
 
-    &:hover,
-    &:active {
-      background-color: $datepicker-bg-hover;
-    }
+    &:not(.is-unavailable):not(:disabled) {
+      &:before {
+        background-color: transparent;
+        border: var(--border-width) solid transparent;
+        border-radius: 2.5rem;
+        content: '';
+        height: 2.5rem;
+        left: 50%;
+        pointer-events: none;
+        position: absolute;
+        top: 50%;
+        transform: translate(-50%, -50%);
+        transition: background-color var(--animation-normal);
+        width: 2.5rem;
+      }
 
-    &:focus {
-      z-index: layer('base');
-    }
+      &.is-current:before {
+        border-color: var(--datepicker-border-color);
+      }
 
-    &.is-current {
-      background-color: $datepicker-today-bg;
-      color: $datepicker-today-color;
-      font-weight: $datepicker-today-font-weight;
+      &.is-selected {
+        color: var(--datepicker-text-color-xlight);
+        font-weight: var(--font-weight-bold);
 
-      &:hover,
-      &:active {
-        background-color: $datepicker-today-bg;
+        &:before {
+          background-color: var(--datepicker-active-color);
+        }
       }
-    }
 
-    &.is-selected {
-      background-color: $datepicker-selected-bg;
-      box-shadow: 0 0 0 var(--border-width) $datepicker-selected-bg;
-      color: $datepicker-selected-color;
-      font-weight: $datepicker-selected-font-weight;
+      &.is-faded {
+        background-color: var(--datepicker-bg-color-dark);
+        color: var(--datepicker-text-color-light);
+      }
+    }
 
+    &:not(.is-selected):not(.is-unavailable):not(:disabled) {
       &:hover,
       &:active,
       &:focus {
-        background-color: $datepicker-selected-bg;
+        color: var(--datepicker-active-color);
+        font-weight: var(--font-weight-bold);
+
+        &:before {
+          background-color: var(--datepicker-highlight-color);
+        }
       }
     }
 
-    &.is-faded {
-      background-color: $datepicker-faded-bg;
-      color: $datepicker-faded-color;
-
-      &:hover,
-      &:active {
-        background-color: $datepicker-faded-bg-hover;
-      }
+    &:focus {
+      z-index: layer('base');
```

Comment:
Dit doet me eraan denken dat we ons z-index trucje ook best eens documenteren.

Reply:
https://jira.antwerpen.be/browse/UDH-156

Ik heb er een ticket voor aangemaakt. Dan vergeten we het niet. 🙂